trollflow2/launcher.py

```diff
@@ -121,6 +125,23 @@ def check_results(produced_files, start_time, exitcode):
                      f"{elapsed!s}", extra={"time": elapsed})
 
 
+def _check_file(saved_file):
+    file_scheme = urlsplit(saved_file).scheme
+    if file_scheme in ('', 'file'):
+        return _check_local_file(saved_file)
+    if file_scheme == 's3':
+        from trollflow2.plugins.s3 import _check_s3_file
```

Comment:
this should be a public method...

Reply:
The `_check_file()` or `_check_s3_file()`?

Reply:
`_check_s3_file` since it's from another module

Reply:
Ah, ok. Thought it would be more in-line with `_check_local_file()`. `_check_s3_file()` is changed to public in 83132af